src/elements/session-element.html

```diff
@@ -148,7 +148,7 @@
 
       <div class="session-header" layout horizontal justified>
         <div flex>
-          <h3 class="session-title">[[session.title]]</h3>
+          <h2 class="session-title">[[session.title]]</h2>
```

Comment:
Headings should be hierarchical - an h3 shouldn't follow an h1. Otherwise the structure is unclear to non-visual users. https://webaim.org/techniques/semanticstructure/#contentstructure